src/mlpack/core/data/imputer.hpp

```diff
@@ -68,7 +68,7 @@ class Imputer
   //! Get the strategy
   const StrategyType& Strategy() const { return strategy; }
 
-  //! Modify the given given strategy (be careful!)
+  //! Modify the given strategy (be careful!)
```

Comment:
Can you remove the ``be careful`` part?

Reply:
I am not so sure what the author actually wanted to indicate by the "be careful !" part but I think as the whole imputation methods depends on the strategy/type of imputation we want to apply to fill the voids so modifying it may lead to some serious issues but on the same hand as in c++ modifying such things anywhere causes some issues so I personally think that "be careful !" is not making a lot of sense here and still if we wanted to alert the programmer then we can keep it untouched or can write "only if you are sure what you are doing" instead of "be careful !"

Reply:
Due to some problem from my side I am closing this PR and created a new PR for all documentation correction here #1789